server/package.json

```diff
@@ -18,6 +18,7 @@
     "npm": "10.8.2"
   },
   "dependencies": {
+    "@ant-design/nextjs-registry": "^1.0.1",
```

Comment:
Remove this, we're not using antd anymore

Reply:
removed